Sprint-3/revise/investigate/find.js

```diff
@@ -20,6 +20,29 @@ console.log(find("code your future", "z"));
 // Pay particular attention to the following:
 
 // a) How the index variable updates during the call to find
+/*Answer:
+1-The index variable starts at 0 and is used to track the position of the current character being checked in the string.
+2-After each iteration of the loop, index is incremented by 1 (index++), moving it to the next character in the string.
+3-The loop continues checking each character until it either finds the target character or reaches the end of the string.
+4-For example, when searching for "u" in "code your future", index starts at 0 and keeps increasing until it finds "u" at index 10.
```

Comment:
The explanation is very clear, but "u" is  first found at an index which is not 10